ProjFS.Mac/PrjFSKext/VirtualizationRoots.cpp

```diff
@@ -144,7 +144,9 @@ VirtualizationRootHandle VirtualizationRoot_FindForVnode(
     
     VirtualizationRootHandle rootHandle = RootHandle_None;
     errno_t error = 0;
-    
+
+    FsidInode vnodeFsidInode = Vnode_GetFsidAndInode(vnode, context, false);
```

Comment:
Quick marker that this was to demonstrate the unit test failure and will be moved back.

Reply:
I've removed that commit in the latest push.